test/java/seedu/addressbook/parser/NameTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void varyingCapitalTest() throws IllegalValueException {
+        Name name1 = new Name("ABCDEFGH");
+        Name name2 = new Name("AbCdEfgH");
+        
+        assertEquals(true, name1.isSimilar(name2));
```

Comment:
do look up the alternatives to `assertEquals(true|false, ...)`: `assertTrue` and `assertFalse`